src/Bicep.IO/Bicep.IO.csproj

```diff
@@ -8,6 +8,7 @@
 
   <ItemGroup>
     <PackageReference Include="System.IO.Abstractions" Version="21.1.3" />
+    <PackageReference Include="System.Memory.Data" Version="6.0.0" />

```

Comment:
>6.0.0 [](http://example.com/codeflow?start=60&length=5)

Any reason not to take 8.0.1 or 9.0.0?

Reply:
Good question...the package was added by VS automatically via completion. Not sure why the old version is used. Let me bump the version.

Reply:
Ah it's because `Azure.Core` has a transitive dependency on `System.Memory.Data 6.0.0`, so VS choosed the local version. Let's keep the current version to avoid any incompatible issues since `Azure.Core` is used by `Bicep.Core`.